### PR TITLE
Make main default branch (and remove gitwash)

### PR DESCRIPTION
**Pull request by jarrodmillman**

### CODE REVIEW

**Review by stefanv — APPROVED** [review]
- skimage script
- note about root_doc for sphinx

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -196,14 +191,14 @@ For a more detailed discussion, read these :doc:`detailed documents
       where 123 is the issue number.
 
 
-Divergence from ``upstream master``
+Divergence from ``upstream main``
 -----------------------------------
```
Comment (stefanv) [inline CONTRIBUTING.rst:195]: Header size

**Review by stefanv — APPROVED** [review]

File: CONTRIBUTING.rst (hunk 1 above)
Comment (jarrodmillman) [reply CONTRIBUTING.rst]: Fixed.

**Review by rossbar — APPROVED** [review]
LGTM - might be worth mentioning the removal of gitwash in the commit msg when merged just so it's discoverable in the history.

A final grep for "master" on this branch didn't turn up any missing instances for me :+1:

**Pull request merged**